Dockerfile

```diff
@@ -0,0 +1,34 @@
+# ==== BUILD =====
+# Use NodeJS v18.x.x with Alpine for a slim image
+FROM node:18-alpine AS build
+LABEL authors="SmashGrade"
+
+# Set the working directory
+WORKDIR /react-docker/
+
+# Set environment variables
+ENV VITE_BACKEND_API_URL=http://localhost:3000
```

Comment:
```suggestion
```
This would overwrite the `VITE_BACKEND_API_URL` env var.

Reply:
Would it actually? Does it work? But you are right I can remove it, since it is handled by the env files. Removed it.